crates/router/tests/connectors/selenium.rs

```diff
@@ -398,6 +398,53 @@ pub trait SeleniumTest {
         pypl_actions.extend(actions);
         self.complete_actions(&c, pypl_actions).await
     }
+    async fn make_clearpay_payment(
+        &self,
+        c: WebDriver,
+        url: &str,
+        actions: Vec<Event<'_>>,
+    ) -> Result<(), WebDriverError> {
+        self.complete_actions(
+            &c,
+            vec![
+                Event::Trigger(Trigger::Goto(url)),
+                Event::Trigger(Trigger::Click(By::Id("card-submit-btn"))),
+            ],
+        )
+        .await?;
+        let (email, pass) = (
+            &self
+                .get_configs()
+                .automation_configs
+                .unwrap()
+                .clearpay_email
+                .unwrap(),
+            &self
+                .get_configs()
+                .automation_configs
+                .unwrap()
+                .clearpay_pass
+                .unwrap(),
+        );
+        let mut clearpay_actions = vec![
+            Event::Trigger(Trigger::Sleep(3)),
+            Event::EitherOr(
+                Assert::IsPresent("Review your order | Clearpay"),
+                vec![Event::Trigger(Trigger::Click(By::ClassName("ai_az")))],
+                vec![
+                    Event::Trigger(Trigger::SendKeys(By::ClassName("n8_fl"), email)),
+                    Event::Trigger(Trigger::Click(By::ClassName("ai_az"))),
+                    Event::Trigger(Trigger::Sleep(3)),
+                    Event::Trigger(Trigger::SendKeys(By::ClassName("n8_fl"), pass)),
+                    Event::Trigger(Trigger::Click(By::ClassName("ai_az"))),
+                    Event::Trigger(Trigger::Sleep(10)), //Time needed for login
```

Comment:
Can't we use css identifier or reduce sleep time ?

Reply:
I tried with different timestamp, but it seems anything smaller than this was giving failure since logging in takes a lot of time